_layouts/event.html

```diff
@@ -68,89 +68,142 @@
       {% assign bannerImgAttrVals = 'banner-img' | split: ' ' -%}
       {% assign bannerImgName = 'events/' | append: page.slug | append: '/banner' -%}
       {% capture bannerImg -%}
-        {% include img.html
-                   img=bannerImgName
-                   alt=page.title
-                   attrKeys=bannerImgAttrKeys
-                   attrVals=bannerImgAttrVals
-         %}
-      {%- endcapture -%}
-      {% if bannerImg -%}
-      <div id="banner" class="banner{% if page.bannerFill %} banner-backfill{% endif %}">
-        <div class="container">
-          {{ bannerImg }}
+      {% include img.html
```

Comment:
AFAIK, you didn’t actually change anything in this block (line 71-97), please revert the indents